### PR TITLE
Create _index.md

### PR DESCRIPTION
**Pull request by Heechberri**

### CODE REVIEW

File: content/authors/Jiayi-Zhang/_index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- Vae Zhang Jiayi
+
+bio: 
+
+education:
+  courses:
+  - course: Ph.D.Candidate, Cradle
+    institution: NTU
+    year: 2020 -
+  - course: Bachalor of Psychology (Hons)
```
Comment (DominiqueMakowski) [inline content/authors/Jiayi-Zhang/_index.md:12]: Hi, thanks. I think there's a typo here

File: content/authors/Jiayi-Zhang/_index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    institution: State University of New York, Buffalo
+    year: 2012 -2015
+
+email: "G200002@e.ntu.edu.sg"
+interests:
+- Muliti-model neuroimaging
```
Comment (DominiqueMakowski) [inline content/authors/Jiayi-Zhang/_index.md:18]: and there

**Comment by Heechberri** [comment]
Hi,

Thanks for pointing them out :)

**Comment by Tam-Pham** [comment]
@Heechberri Hi Jiayi, thank you for making the changes. But your avatar is currently missing in your folder. Could you please upload a photo of yourself and create another PR? :)

**Pull request merged**